src/Promitor.Agents.Scraper/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -236,7 +237,12 @@ private static void AddOpenTelemetryCollectorMetricSink(string collectorUri, str
                     {
                         metricsBuilder.SetResourceBuilder(resourceBuilder)
                                       .AddMeter("Promitor.Scraper.Metrics.AzureMonitor")
-                                      .AddOtlpExporter(options => options.Endpoint = new Uri(collectorUri));
+                                      .AddOtlpExporter(options =>
+                                          {
+                                              options.Endpoint = new Uri(otelConfiguration.CollectorUri);
+                                              options.Protocol = otelConfiguration.Protocol;
```

Comment:
Only assign when configured
```suggestion
                                           if(otelConfiguration.Protocol != null)
                                           {
                                              options.Protocol = otelConfiguration.Protocol;
                                           }
```